source/Playnite.DesktopApp/Themes/Desktop/Default/CustomControls/ComboBoxList.xaml

```diff
@@ -47,7 +47,27 @@
                     <Border Background="{DynamicResource PopupBackgroundBrush}"
                             BorderThickness="{DynamicResource PopupBorderThickness}"
                             BorderBrush="{DynamicResource PopupBorderBrush}">
-                        <ItemsControl x:Name="PART_ItemsPanel" />
+                        <Grid>
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="auto" />
+                                <RowDefinition Height="*" />
+                            </Grid.RowDefinitions>
+
+                            <Grid Grid.Row="0" Margin="5"
+                                  Visibility="{TemplateBinding UseSearchBox, Converter={StaticResource BooleanToVisibilityConverter}}">

```

Comment:
This must be set from code behind, themes should not be able to change/break this..